src/app/components/Tokens/TokenLinkWithIcon.tsx

```diff
@@ -17,9 +17,9 @@ export const TokenLinkWithIcon: FC<{
   return (
     <Box sx={{ display: 'flex', alignItems: 'center', gap: 4 }}>
       {metadata?.icon ? (
-        <img src={metadata.icon} alt="" width={28} />
+        <img src={metadata.icon} alt="" width={40} style={{ maxHeight: 32, margin: '-4px 0' }} />
       ) : (
-        <InitialsAvatar name={metadata?.name || name || address.slice(2, 4)} size={28} />
+        <InitialsAvatar name={metadata?.name || name || address.slice(2, 4)} width={40} height={28} />
```

Comment:
shouldn't size of a token icon be the same as "avatar icon"?  Currently avatars are smaller